Prace_domowe/Praca_domowa3/Grupa2/Pawel_Niewiadowski/PD3.ipynb

```diff
@@ -0,0 +1,999 @@
+{
```

Comment:
<p>Uważam, że w tym wypadku nie jest to optymalne podejście. Pozbyłeś się dużej liczby być może wartościowych obserwacji. Niezbalansowany target bywa problemem, ale może sobie z nim radzić na różne inne sposoby -- na przykład wybierając zbalansowany zbiór testowy, lub używając miary oceny klasyfikatora która dobrze sobie radzi z niezbalansowanymi zbiorami (np. AUC).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>